### PR TITLE
[SPDBT-2968] Intergration test for controlling member crc repository

### PR DESCRIPTION
**Pull request by esdd1995**
# Description

### CODE REVIEW

File: src/Spd.Resource.Repository.IntegrationTest/ControllingMemberCrcRepositroyTest.cs [hunk 1]
```diff
@@ -0,0 +1,396 @@
...
+using Spd.Resource.Repository.PersonLicApplication;
+using Spd.Utilities.Dynamics;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Spd.Resource.Repository.IntegrationTest;
+public class ControllingMemberCrcRepositroyTest : IClassFixture<IntegrationTestSetup>
```
Comment (carolcarpenter) [inline src/Spd.Resource.Repository.IntegrationTest/ControllingMemberCrcRepositroyTest.cs:16]: Spelling mistake in class and filename 'Repository'

File: src/Spd.Resource.Repository.IntegrationTest/ControllingMemberCrcRepositoryTest.cs [hunk 2]
```diff
@@ -0,0 +1,396 @@
...
+    }
+
+    private async Task CleanUpObject(Guid? contactId, Guid? accountid, Guid? bizlicAppId, Guid? bizContactId, Guid? controllingMemberAppId)
+    {
+        spd_application? appToRemove = _context.spd_applications.Where(a => a.spd_applicationid == controllingMemberAppId).SingleOrDefault();
+
+        if (appToRemove != null)
+        {
+            _context.DeleteObject(appToRemove);
+            await _context.SaveChangesAsync();
```
Comment (peggy-quartech) [inline src/Spd.Resource.Repository.IntegrationTest/ControllingMemberCrcRepositoryTest.cs:364]: do not save changes everytime. 
Remove it. at the end of function, add saveChanges.

**Review by peggy-quartech — CHANGES_REQUESTED** [review]

File: src/Spd.Resource.Repository.IntegrationTest/ControllingMemberCrcRepositoryTest.cs (hunk 2 above)
Comment (esdd1995) [reply src/Spd.Resource.Repository.IntegrationTest/ControllingMemberCrcRepositoryTest.cs:364]: this way it causes exception

Comment (peggy-quartech) [reply src/Spd.Resource.Repository.IntegrationTest/ControllingMemberCrcRepositoryTest.cs:364]: what exception?

**Review by peggy-quartech — APPROVED** [review]

**Pull request merged**